addons/account/models/account_move.py

```diff
@@ -4181,8 +4199,28 @@ def _cron_account_move_send(self, job_count=10):
                 else:
                     raise
 
+            # Retrieve res.partner that executed the Send & Print wizard
+            sp_partner_ids = set(moves.mapped(lambda move: move.send_and_print_values and move.send_and_print_values['sp_partner_id']))
+            sp_partners = self.env['res.partner'].browse(sp_partner_ids)
+            moves_map = {
+                partner: moves.filtered(lambda m: m.send_and_print_values['sp_partner_id'] == partner.id)
+                for partner in sp_partners
+            }
+
             self.env['account.move.send']._process_send_and_print(moves)
 
+            notifications = []
+            for partner, partner_moves in moves_map.items():
+                partner_moves_error = partner_moves.filtered(lambda m: m.send_and_print_values and m.send_and_print_values.get('error'))
```

Comment:
`send_and_print_values` is always set here
```suggestion
                partner_moves_error = partner_moves.filtered(lambda m: m.send_and_print_values.get('error'))
```

Reply:
Yes for the one above, but no for this one: the `_process_send_and_print` changes the `send_and_print_values` if everything is successful to False.

https://github.com/odoo/odoo/pull/149717/files#diff-69133af9cc727cff6007b627417e589681d05648c0902449a6e28506d0f422aeR682